bundles/AdminBundle/Controller/Admin/TargetingController.php

```diff
@@ -37,6 +37,11 @@ class TargetingController extends AdminController implements KernelControllerEve
 {
     // RULES
 
+    private function correctName(string $name): string
+    {
+        return preg_replace('/[#\?\*\:\\\\<\>\|"%&@=;\+]/', '-', $name);
```

Comment:
This much escaping is not necessary here, the following should be sufficient:

```suggestion
        return preg_replace('/[#?*:\\\\<>|"%&@=;+]/', '-', $name);
```

Reply:
You're right, thanks!